### DISCUSSION TITLE
resize split without mouse

### DISCUSSION THREAD

**Discussion by comod** [JetBrains/ideavim#430] [Q&A] [answered]
how to resize split without mouse? usually vim  commands doesnt work and even idea shortcuts for resizing windows doesnt work aswell :-(

**Comment by Adir-Shemesh** [comment, marked as answer]
The [Tab Shifter](https://plugins.jetbrains.com/plugin/7475-tab-shifter) IDE plugin introduces actions to stretch splits - you can map those in your ideavimrc or in the IDE keymap settings
